ops/scripts/deployer.sh

```diff
@@ -18,5 +18,24 @@ curl \
 
 yarn run deploy
 
+function envSet() {
+    VAR=$1
+    export $VAR=$(cat ./dist/dumps/addresses.json | jq -r ".$2")
+}
+
+# set the address to the proxy gateway if possible
```

Comment:
How does this work? It looks like the dump is created after the `envSet` is used to set the variables. Also small nit, you could technically do `local VAR=$1` inside the bash function to prevent the variable from being set globally, ie if you reference `VAR` outside of the function then it'll be present but this is a small script and doesn't really matter

Reply:
envSet is used to set the variables from the L1 addresses (addresses.json). So the L2 dump needs to be created after the L1 contracts are deployed. Honestly I just copy/pasted the envSet function from geth.sh.

Reply:
Oh i see, I saw it in `dist/dumps` which made me confused